kinode/packages/app_store/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "App Store",
+    "description": "A package manager + app store.",
+    "image": "",
+    "properties": {
+        "package_name": "app_store",
+        "current_version": "0.2.0",
+        "publisher": "sys",
+        "mirrors": [],
+        "code_hashes": {
+            "0.2.0": "",
+            "0.1.0": ""
```

Comment:
since only supported versions should be listed here, should just be 0.3.0 as only element?